x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/BooleanBlockBuilder.java

```diff
@@ -180,21 +181,33 @@ public BooleanBlockBuilder mvOrdering(Block.MvOrdering mvOrdering) {
 
     @Override
     public BooleanBlock build() {
-        finish();
-        BooleanBlock block;
-        if (hasNonNullValue && positionCount == 1 && valueCount == 1) {
-            block = blockFactory.newConstantBooleanBlockWith(values[0], 1, estimatedBytes);
-        } else {
-            if (values.length - valueCount > 1024 || valueCount < (values.length / 2)) {
-                values = Arrays.copyOf(values, valueCount);
-            }
-            if (isDense() && singleValued()) {
-                block = blockFactory.newBooleanArrayVector(values, positionCount, estimatedBytes).asBlock();
+        try {
+            finish();
+            BooleanBlock theBlock;
+            if (hasNonNullValue && positionCount == 1 && valueCount == 1) {
+                theBlock = blockFactory.newConstantBooleanBlockWith(values[0], 1, estimatedBytes);
             } else {
-                block = blockFactory.newBooleanArrayBlock(values, positionCount, firstValueIndexes, nullsMask, mvOrdering, estimatedBytes);
+                if (values.length - valueCount > 1024 || valueCount < (values.length / 2)) {
+                    values = Arrays.copyOf(values, valueCount);
+                }
+                if (isDense() && singleValued()) {
+                    theBlock = blockFactory.newBooleanArrayVector(values, positionCount, estimatedBytes).asBlock();
+                } else {
+                    theBlock = blockFactory.newBooleanArrayBlock(
+                        values,
+                        positionCount,
+                        firstValueIndexes,
+                        nullsMask,
+                        mvOrdering,
+                        estimatedBytes
+                    );
+                }
             }
+            built();
+            return theBlock;
+        } catch (CircuitBreakingException e) {
```

Comment:
I'm ok with this, to get this PR in, but I'd like to find the usage sites and add an explicit close. But that can follow

Reply:
I agree with you, I'll merge this and follow up immediately